extensions/http/jetty-micrometer/src/main/java/org/eclipse/dataspaceconnector/extension/jetty/JettyMicrometerConfiguration.java

```diff
@@ -33,6 +34,18 @@ public JettyMicrometerConfiguration(MeterRegistry registry) {
 
     @Override
     public void accept(ServerConnector connector) {
-        connector.addBean(new JettyConnectionMetrics(registry, connector, Tags.empty()));
+        var tags = Tags.empty();
+
+        if (connector.getPort() != 0) {
+            tags = tags.and("jetty_port", String.valueOf(connector.getPort()));
+        } else {
+            tags = tags.and("jetty_port", String.valueOf(connector.getLocalPort()));
+        }
+
+        if (connector.getName() != null) {
```

Comment:
This condition is always true as the connector name is taken by the `PortMapping` name, and that's in turn always not null by design.